README.md

```diff
@@ -47,7 +47,7 @@ After you're finished please remove all the comments and instructions!
   - [Usage](#usage)
   - [Run tests](#run-tests)
```

Comment:
- [ ] There is no sound in the audio recording. Please record a new video.

Reply:
Implemented thank you for reviewing. :+1: